db/migrate/20150203151605_cleanup_old_fields.rb

```diff
@@ -22,9 +22,17 @@ def change
     remove_column :courses, :late_penalty_old, :float
     remove_column :courses, :version_penalty_old, :float
 
-    remove_column :submissions, :tweak_old, :float
-    remove_column :submissions, :ignored_old, :boolean
-    remove_column :submissions, :absolute_tweak, :boolean
+    if column_exists? :submissions, :tweak_old
+        remove_column :submissions, :tweak_old, :float
+    end
+    
+    if column_exists? :submissions, :ignored_old
+        remove_column :submissions, :ignored_old, :boolean
+    end
+
+    if column_exists? :submissions, :absolute_tweak  
+        remove_column :submissions, :absolute_tweak, :boolean
+    end
```

Comment:
I don't want it to run. Those columns are already removed.
On Sat, Feb 14, 2015 at 3:02 PM Daniel Bucci <notifications@github.com>
wrote:

> In db/migrate/20150203151605_cleanup_old_fields.rb
> <https://github.com/autolab/Autolab/pull/276#discussion_r24716833>:
>
> > @@ -22,9 +22,17 @@ def change
> >      remove_column :courses, :late_penalty_old, :float
> >      remove_column :courses, :version_penalty_old, :float
> >
> > -    remove_column :submissions, :tweak_old, :float
> > -    remove_column :submissions, :ignored_old, :boolean
> > -    remove_column :submissions, :absolute_tweak, :boolean
> > +    if column_exists? :submissions, :tweak_old
> > +        remove_column :submissions, :tweak_old, :float
> > +    end
> > +
> > +    if column_exists? :submissions, :ignored_old
> > +        remove_column :submissions, :ignored_old, :boolean
> > +    end
> > +
> > +    if column_exists? :submissions, :absolute_tweak
> > +        remove_column :submissions, :absolute_tweak, :boolean
> > +    end
>
> It won't run if you change an old migration file that's been run. You'd
> need to rollback, which introduces a bunch of potential problems.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/autolab/Autolab/pull/276/files#r24716833>.
>